checkbox-provider-ce-oem/bin/graphics_test.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+test_ubuntu_frame_launching() {
+    if pgrep -if "ubuntu-frame" > /dev/null; then
+        echo "The ubuntu-frame is active"
+        echo "No need to bring it up again"
+        echo
+        echo "journal log of ubuntu frame:"
+        journalctl -b 0 | grep "ubuntu-frame"
+        touch "$PLAINBOX_SESSION_SHARE/ubuntu-frame-is-activating"
+        exit 0
+    else
+        echo "ubuntu-frame is not active"
+        echo "Activating now..."
+        echo
+        # Expecting exit code 124 from the 'timeout' command
+        # Capture any exit codes other than 0 and 124; return exit code 1 in such cases
+        timeout 20s ubuntu-frame || ( [[ $? -eq 124 ]] && \
+        echo -e "\nPASS: Timeout reached without any failures detected." )
+    fi
+}
+
+test_glmark2_es2_wayland() {
+    CMD="env XDG_RUNTIME_DIR=/run/user/0 graphics-test-tools.glmark2-es2-wayland"
+    OUTPUT=""
+    EXIT=0
+    if [ -f "$PLAINBOX_SESSION_SHARE/ubuntu-frame-is-activating" ]; then
```

Comment:
I don't think we need the 'ubuntu-frame-is-activating' file to check the ubuntu-frame is active or not.